README.md

```diff
@@ -310,23 +310,38 @@ The NeMo Framework can be installed in a variety of ways, depending on
 your needs. Depending on the domain, you may find one of the following
 installation methods more suitable.
 
-- Conda / Pip - Refer to [Conda](#conda) and [Pip](#pip) for
-  installation instructions.
+- [Conda / Pip](#conda--pip): Install NeMo-Framework with native Pip into a virtual environment.
+  - Perfect to explore NeMo on any supported platform.
   - This is the recommended method for ASR and TTS domains.
-  - When using a Nvidia PyTorch container as the base, this is the
-      recommended method for all domains.
-- Docker Containers - Refer to [Docker containers](#docker-containers)
-  for installation instructions.
-  - NeMo Framework container -
-      [nvcr.io/nvidia/nemo:24.05]{.title-ref}
-- LLMs and MMs Dependencies - Refer to [LLMs and MMs
-    Dependencies](#install-llms-and-mms-dependencies) for installation
-    instructions.
+  - Limited feature-completeness for other domains.
+- [NGC PyTorch container](#ngc-pytorch-container): Install NeMo-Framework from source with feature-completeness into a highly optimized container.
+  - Perfect for customers that seek flexibility.
+  - Provides most flexibility for container of your choice.
+- [NGC NeMo container](#ngc-nemo-container): Ready-to-go solution of NeMo-Framework
+  - Perfect for customers that seek highest performance.
+  - Insides all dependencies and NeMo libraries such as NeMo-Aligner, NeMo-Curator, and more.
 
-**Important: We strongly recommended that you start with a base NVIDIA
-PyTorch container: nvcr.io/nvidia/pytorch:24.02-py3.**
+### Support matrix
 
-### Conda
+NeMo-Framework provides tiers of support based on OS / Platform and mode of installation. Please refer the following overview of support levels:
+
+- Fully supported: Max performance and feature-completeness.
+- Limited supported: Perfect to explore NeMo.
+- No support yet: In development.
+- Deprecated: Support has reached end of life.
+
+Please refer to the following table for current support levels:
+
+| OS / Platform              | Pip-install     | Source into NGC container |
```

Comment:
Pip-install is a bit ambiguous. Let's use Install from PyPi or something like that.